File: app/src/main/java/com/raiffeisen/backend/app/repository/model/Account.java
package com.raiffeisen.backend.app.repository.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Account {

    private Long accountId;
    private Long userId;
    private BigDecimal balance;
    private boolean isForSelf;
    private String cardName;
}

File: Android/app/src/main/java/com/github/ivanshafran/raif/data/model/Account.kt
package com.github.ivanshafran.raif.data.model

import com.google.gson.annotations.SerializedName
import java.math.BigDecimal

class Account(
    @SerializedName("balance")
    val balance: BigDecimal,
    @SerializedName("cardname")
    val cardName: String
)
